raiden/network/transport/matrix/transport.py

```diff
@@ -1198,11 +1203,7 @@ def _get_room_ids_for_address(
         address_hex: AddressHex = to_checksum_address(address)
```

Comment:
is hex here the same as checksum? or should you rename `address_hex` to `address_checksum ?`

Reply:
to_checksum_address returns hex, so in a way yeah it's the same

Reply:
ok, I always thought that checksum is a subset of hex :)

Reply:
But there is a `ChecksumAddress` type, so you're right